### ISSUE TITLE
Corrupts Saves for the 6th floor and beyond.

### ISSUE THREAD

**Issue by aknight2015** [01y/bstone-vita#3]
Both Aliens of Gold and Planet Strike saves are corrupted when you try to save on the 6th level or later. I'm using 0.3 release. I've tried multiple times and it always has the same result.

**Comment by Andiweli** [comment]
Have this from Stage 8 onwards. Level 7 can be loaded without problems. Level 8 is always corrupt when loaded.

**Comment by aknight2015** [comment]
> Have this from Stage 8 onwards. Level 7 can be loaded without problems. Level 8 is always corrupt when loaded.

Where's the save file?

**Comment by Andiweli** [comment]
[BStone.zip](https://github.com/01y/bstone-vita/files/14001802/BStone.zip)

Here my saves... might help debugging.

**Comment by aknight2015** [comment]
Thanks.

**Comment by Andiweli** [comment]
Any updates on this issue?

**Comment by aknight2015** [comment]
Nope. My guess is the developer went "Good enough" and moved on to other projects.
